### PR TITLE
PENG-3037  `reg` claim split

### PR DESCRIPTION
**Pull request by inesea**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -22,9 +22,14 @@ EVENT_NAME=Some descriptive name
...
-# Should contain a group that specifies the known dynamic part of the URL
-REGEXES=streaming-simulator.*(\/.*\.ts|\/.*\.m3u8|.*\.mp4), another-host.*(\/.*\.ts|\/.*\.m3u8|.*\.mp4)
+# List of regexes to match the stream manifest URLs. It will be used to create a single context
+# for all playlists and segments from the same manifest. Should contain a group that specifies the
+# known dynamic part of the URL. 
+MANIFEST_REGEXES=streaming-simulator.*(\/.*\.m3u8|.*\.mp4), another-host.*(\/.*\.m3u8|.*\.mp4)
+
+# List of regexes to match the playlist/segment URL which identifies the same
+# stream/segment when the URL has different sources
+QUALITY_REGEXES=some-quality.*(\/.*\.ts), another-host.*(\/.*\.ts)
```
[the remark this answers is not in the document]
Comment (inesea) [reply .env.example:32]: Good point

File: src/jwt/create-jwt-regex.js [hunk 2]
```diff
@@ -55,4 +61,4 @@ const jwt = jsonwebtoken.sign(payload, privateKey, {
     keyid: keyId
 });
 
-console.log(`JWT string: ${jwt}`);
+console.log(`JWT: ${jwt}}`);
```
Comment (josepabellana) [inline src/jwt/create-jwt-regex.js:64]: I think theres a small typo adding two } if I'm not wrong

Comment (inesea) [reply src/jwt/create-jwt-regex.js]: Ouch yes fixed

File: .env.example (hunk 1 above)
[the remark this answers is not in the document]
Comment (inesea) [reply .env.example]: I added `.m3u8` to signify the playlist urls, but regarding others I think we don't necessarily need to give an exhaustive list in the example

**Review by josepabellana — APPROVED** [review]

File: src/jwt/create-jwt-regex.js (hunk 2 above)
Comment (josepabellana) [reply src/jwt/create-jwt-regex.js]: perfect

File: .env.example (hunk 1 above)
[the remark this answers is not in the document]
Comment (josepabellana) [reply .env.example]: thanks

**Pull request merged**